docs/contributing.md

```diff
@@ -35,6 +35,38 @@ Adding examples is quite straight forward, so don't feel shy! Here's how to do i
 
 Aaaand you are done! :tada: 
 
+### New Generators
+Generators sits as the core of Modelina, which frames the core concepts of what you can generate. Therefore it's also no small task to create a new one, so dont get discourage, we are here to help you! 
+
+To make it easier to contribute a new generator, and to avoid focusing too much of the internals of Modelina, we created a template generator to get you started. If you encounter discreprencies with the following guide or templates, make sure to raise it as an issue so it can be fixed!
+
```

Comment:
I'd throw a link into Slack here, to encourage potential contributors to join/chat before diving into their implementation. I had to rifle around looking for IRC/chat channel.

Reply:
Agree 👍

Reply:
I ended up adding the following
```
If you have any questions, are unsure how your use-case fits in, or want something clarified, don't hesitate to [reach out on slack](https://asyncapi.com/slack-invite).
```